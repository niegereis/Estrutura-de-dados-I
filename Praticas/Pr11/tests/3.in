3
5 4 2 2 5 3
Alice 1629405 71 92.2
Miguel 1929056 82 87.1
Sophia 2926202 75 115.3
