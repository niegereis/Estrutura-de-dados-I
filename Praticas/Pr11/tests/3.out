[1929056] Miguel - 82 (87.1)
[2926202] Sophia - 75 (115.3)
[1629405] Alice - 71 (92.2)
